README.md

```diff
@@ -86,8 +89,13 @@ with the Conda installation (sections [1a](#1a-anaconda) and [1b](#1b-starting-f
 
 1. Binary installation (Anaconda)
 
+The binary distribution of Sherpa 4.8.1 only supports Python 2.7.
+
 2. Source build (from a source tarball or the GitHub repository)
 
+A source build is - at present - the *only* way to use Sherpa 
```

Comment:
Well, we'll have to change a few other things in the file then, so hopefully we'll notice!